nat-lab/tests/test_pinger.py

```diff
@@ -57,11 +57,13 @@ async def test_ping_pong() -> None:
 
         # Send PING type message
         async with run_async_context(client_alpha.receive_ping()) as ping:
+            await client_alpha.wait_for_listen_port_ready("udp", 5000, "python3")
```

Comment:
Minor: worth extracting 5000 into a common variable since it's being used through the file